core/crypto/validator_impl.go

```diff
@@ -176,10 +163,10 @@ func (validator *validatorImpl) initCryptoEngine() (err error) {
 		nil, validator.enrollChainKey.(*ecdsa.PrivateKey),
 	)
 	if err != nil {
-		return
+		return err
```

Comment:
It is not necessary to specify err here given that the return variable has been named.